applications/launchpad/backend/src/api/wallet_api.rs

```diff
@@ -67,7 +67,12 @@ pub async fn wallet_balance() -> Result<WalletBalance, String> {
 pub async fn wallet_events(app: AppHandle<Wry>) -> Result<(), String> {
     info!("Setting up event stream");
     let mut wallet_client = GrpcWalletClient::new();
-    let mut stream = wallet_client.stream().await.map_err(|e| e.chained_message()).unwrap();
+
+    let mut stream = match wallet_client.stream().await.map_err(|e| e.chained_message()) {
+        Ok(s) => s,
+        Err(_err) => return Err("wallet is not running".to_string()),
+    };
```

Comment:
```suggestion
    let mut stream = match wallet_client.stream().await.map_err(|e| e.chained_message())?;
```
Just let the true cause of the error bubble up